src/components/tools/editor/editor_components/editor_elements/Paragraph.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="input-container">
+    <p
+      @focus="select()"
+      class="input"
+      ref="editable"
+      :id="index"
+      contenteditable
+      @input="$emit('input', $refs.editable.innerText)"
+      @keydown.enter.prevent="newParagraph()"
+      @keydown.delete="deleteParagraph()"
+      placeholder="Skriv här..."
+    />
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    value: {
+      type: String,
+      default: null,
+    },
+    index: {
+      type: Number,
+      required: true
+    },
+    newCheck: {
+      type: Boolean,
+      default: false
+    }
+  },
+  mounted() {
+    this.$refs.editable.innerText = this.value
+  },
+  methods: {
+    select() {
+      setTimeout(() => {
```

Comment:
Explain reason for timeout

Reply:
Changed timeout to nextTick as it provides consistency and is better suited for the purpose
Added explanation